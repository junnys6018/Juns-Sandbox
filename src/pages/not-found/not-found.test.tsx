import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import NotFound from './not-found';

test('<NotFound /> snapshot', () => {
    const { asFragment } = render(<NotFound />, { wrapper: MemoryRouter });
    expect(asFragment()).toMatchSnapshot();
});
