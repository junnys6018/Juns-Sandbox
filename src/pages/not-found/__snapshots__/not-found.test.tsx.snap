// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex flex-col min-h-screen"
  >
    <nav
      class="container flex flex-row items-center py-10"
    >
      <a
        class="font-semibold text-3xl mr-auto bg-clip-text text-transparent bg-gradient-to-r from-[#ff0099] to-[#ff9900]"
        href="/"
      >
        Jun's Sandbox
      </a>
      <a
        class="font-medium text-xl text-pink-600"
        href="/"
      >
        Home
      </a>
    </nav>
    <div
      class="text-center mt-[20vh]"
    >
      <p
        class="font-semibold text-pink-600 text-6xl md:text-7xl mb-8 md:mb-11"
      >
        Oops
      </p>
      <p
        class="font-medium text-pink-600 text-3xl md:text-4xl mb-4 md:mb-5"
      >
        Something Went Wrong
      </p>
      <p
        class="font-normal text-neutral-800 text-xl md:text-2xl"
      >
        Error 404 Page Not Found
      </p>
    </div>
    <footer
      class="mt-auto text-center pb-4"
    >
      <div
        class="xl:hidden flex flex-row justify-center gap-4"
      >
        <a
          class="p-2 rounded-full hover:bg-pink-500 hover:text-neutral-50 transition"
          href="https://github.com/junnys6018/Juns-Sandbox"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            fill="none"
            height="1em"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M9 19c-5 1.5-5-2.5-7-3m14 6v-3.87a3.37 3.37 0 0 0-.94-2.61c3.14-.35 6.44-1.54 6.44-7A5.44 5.44 0 0 0 20 4.77 5.07 5.07 0 0 0 19.91 1S18.73.65 16 2.48a13.38 13.38 0 0 0-7 0C6.27.65 5.09 1 5.09 1A5.07 5.07 0 0 0 5 4.77a5.44 5.44 0 0 0-1.5 3.78c0 5.42 3.3 6.61 6.44 7A3.37 3.37 0 0 0 9 18.13V22"
            />
          </svg>
        </a>
        <a
          class="p-2 rounded-full hover:bg-pink-500 hover:text-neutral-50 transition"
          href="https://www.linkedin.com/in/jun-lim-131810180/"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 448 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M100.28 448H7.4V148.9h92.88zM53.79 108.1C24.09 108.1 0 83.5 0 53.8a53.79 53.79 0 0 1 107.58 0c0 29.7-24.1 54.3-53.79 54.3zM447.9 448h-92.68V302.4c0-34.7-.7-79.2-48.29-79.2-48.29 0-55.69 37.7-55.69 76.7V448h-92.78V148.9h89.08v40.8h1.3c12.4-23.5 42.69-48.3 87.88-48.3 94 0 111.28 61.9 111.28 142.3V448z"
            />
          </svg>
        </a>
        <a
          class="p-2 rounded-full hover:bg-pink-500 hover:text-neutral-50 transition"
          href="https://junnys6018.github.io/"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 496 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M336.5 160C322 70.7 287.8 8 248 8s-74 62.7-88.5 152h177zM152 256c0 22.2 1.2 43.5 3.3 64h185.3c2.1-20.5 3.3-41.8 3.3-64s-1.2-43.5-3.3-64H155.3c-2.1 20.5-3.3 41.8-3.3 64zm324.7-96c-28.6-67.9-86.5-120.4-158-141.6 24.4 33.8 41.2 84.7 50 141.6h108zM177.2 18.4C105.8 39.6 47.8 92.1 19.3 160h108c8.7-56.9 25.5-107.8 49.9-141.6zM487.4 192H372.7c2.1 21 3.3 42.5 3.3 64s-1.2 43-3.3 64h114.6c5.5-20.5 8.6-41.8 8.6-64s-3.1-43.5-8.5-64zM120 256c0-21.5 1.2-43 3.3-64H8.6C3.2 212.5 0 233.8 0 256s3.2 43.5 8.6 64h114.6c-2-21-3.2-42.5-3.2-64zm39.5 96c14.5 89.3 48.7 152 88.5 152s74-62.7 88.5-152h-177zm159.3 141.6c71.4-21.2 129.4-73.7 158-141.6h-108c-8.8 56.9-25.6 107.8-50 141.6zM19.3 352c28.6 67.9 86.5 120.4 158 141.6-24.4-33.8-41.2-84.7-50-141.6h-108z"
            />
          </svg>
        </a>
      </div>
      <span
        class="font-medium text-xs text-neutral-700"
      >
        Â© 2022 Designed & Built by Jun Lim 
      </span>
    </footer>
    <div
      class="fixed bottom-4 left-4 hidden xl:flex flex-col items-center justify-center gap-2"
    >
      <a
        class="p-2 rounded-full hover:bg-pink-500 hover:text-neutral-50 transition"
        href="https://github.com/junnys6018/Juns-Sandbox"
        rel="noopener noreferrer"
        target="_blank"
      >
        <svg
          fill="none"
          height="1em"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M9 19c-5 1.5-5-2.5-7-3m14 6v-3.87a3.37 3.37 0 0 0-.94-2.61c3.14-.35 6.44-1.54 6.44-7A5.44 5.44 0 0 0 20 4.77 5.07 5.07 0 0 0 19.91 1S18.73.65 16 2.48a13.38 13.38 0 0 0-7 0C6.27.65 5.09 1 5.09 1A5.07 5.07 0 0 0 5 4.77a5.44 5.44 0 0 0-1.5 3.78c0 5.42 3.3 6.61 6.44 7A3.37 3.37 0 0 0 9 18.13V22"
          />
        </svg>
      </a>
      <a
        class="p-2 rounded-full hover:bg-pink-500 hover:text-neutral-50 transition"
        href="https://www.linkedin.com/in/jun-lim-131810180/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <svg
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 448 512"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M100.28 448H7.4V148.9h92.88zM53.79 108.1C24.09 108.1 0 83.5 0 53.8a53.79 53.79 0 0 1 107.58 0c0 29.7-24.1 54.3-53.79 54.3zM447.9 448h-92.68V302.4c0-34.7-.7-79.2-48.29-79.2-48.29 0-55.69 37.7-55.69 76.7V448h-92.78V148.9h89.08v40.8h1.3c12.4-23.5 42.69-48.3 87.88-48.3 94 0 111.28 61.9 111.28 142.3V448z"
          />
        </svg>
      </a>
      <a
        class="p-2 rounded-full hover:bg-pink-500 hover:text-neutral-50 transition"
        href="https://junnys6018.github.io/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <svg
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 496 512"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M336.5 160C322 70.7 287.8 8 248 8s-74 62.7-88.5 152h177zM152 256c0 22.2 1.2 43.5 3.3 64h185.3c2.1-20.5 3.3-41.8 3.3-64s-1.2-43.5-3.3-64H155.3c-2.1 20.5-3.3 41.8-3.3 64zm324.7-96c-28.6-67.9-86.5-120.4-158-141.6 24.4 33.8 41.2 84.7 50 141.6h108zM177.2 18.4C105.8 39.6 47.8 92.1 19.3 160h108c8.7-56.9 25.5-107.8 49.9-141.6zM487.4 192H372.7c2.1 21 3.3 42.5 3.3 64s-1.2 43-3.3 64h114.6c5.5-20.5 8.6-41.8 8.6-64s-3.1-43.5-8.5-64zM120 256c0-21.5 1.2-43 3.3-64H8.6C3.2 212.5 0 233.8 0 256s3.2 43.5 8.6 64h114.6c-2-21-3.2-42.5-3.2-64zm39.5 96c14.5 89.3 48.7 152 88.5 152s74-62.7 88.5-152h-177zm159.3 141.6c71.4-21.2 129.4-73.7 158-141.6h-108c-8.8 56.9-25.6 107.8-50 141.6zM19.3 352c28.6 67.9 86.5 120.4 158 141.6-24.4-33.8-41.2-84.7-50-141.6h-108z"
          />
        </svg>
      </a>
    </div>
  </div>
</DocumentFragment>
`;
